tests/foreman/cli/test_satellitesync.py

```diff
@@ -400,6 +400,83 @@ def tearDown(self):
         super(ContentViewSync, self).tearDown()
         ssh.command('rm -rf {}/*'.format(self.export_base))
 
+    @run_only_on('sat')
+    @tier3
+    def test_positive_export_import_filtered_cvv(self):
+        """Export Filtered CV version contents in directory and Import them.
+
+        :id: 2992e0ae-173d-4589-817d-1a11455dfc43
+
+        :steps:
+
+            1. Create product and repository with custom contents.
+            2. Sync the repository.
+            3. Create CV with above product.
+            4. Create a filter and filter-rule.
+            5. Publish the above filtered content-view.
+            6. Export Filtered CV version contents to a directory
+            7. Import those contents from some other org/satellite.
+
+        :expectedresults:
+
+            1. Filtered CV version custom contents has been exported to directory
+            2. All The exported custom contents has been imported in org/satellite
+
+        :CaseAutomation: Automated
+
+        :CaseComponent: ContentViews
+
+        :CaseImportance: High
+
+        :CaseLevel: System
+        """
+        exporting_cv_name = gen_string('alpha')
+        exporting_cv, exporting_cvv = ContentViewSync._create_cv(
+            exporting_cv_name, self.exporting_repo, self.exporting_org, False)
+        filter_name = gen_string('alphanumeric')
+        ContentView.filter.create({
+            'name': filter_name,
+            'content-view-id': exporting_cv['id'],
+            'inclusion': 'no',
```

Comment:
If you can go for yes, that would be cool as if we can test the only one rpm that is 'cat'.